adkintunmobile/server/populate/Dockerfile

```diff
@@ -38,7 +38,8 @@ RUN apt-get purge -y \
     git \
     libpq-dev \
     make \
-    wget
+    wget && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
I think this last command could be in a different RUN command, but no sure. Your call